assertj-core/src/test/java/org/assertj/core/error/ShouldBeEqual_Test.java

```diff
@@ -154,7 +166,7 @@ void should_display_multiline_values_nicely_with_comparison_strategy() {
   }
 
   @Test
-  void should_display_multiline_values_nicely_for_ambiguous_representation_for_ambiguous_representation() {
+  void should_display_multiline_values_nicely_with_comparison_strategy_for_ambiguous_representation() {
```

Comment:
Renamed this because it contained `for_ambiguous_representation` twice, and the test in the lines in front also used `..._with_comparison_strategy` as name.